paper-input-behavior.html

```diff
@@ -388,7 +388,7 @@
      * Returns a reference to the input element.
      */
     get inputElement() {
-      return this.$.input;
+      return this.querySelector('.input-element');
```

Comment:
We should try to cache this -- the benefit of `this.$` is that it's cached, and you don't do this query all the time.

Something like this should work:
```
if (!this._inputElement) { 
  this._inputElement = this.querySelector('.input-element');
}
return this._inputElement;
```

Reply:
(can we just do `this.querySelector('input');` and not add the extra useless class?>

Reply:
Actually: can we just assign to `this.$` rather than something else? So:

```
if (!this.$) { 
  this.$ = {}
}
if (!this.$.input) { 
  this.$.input = this.querySelector('input');
}
return this.$.input;
```